src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -257,6 +259,30 @@ public final String getMetaFullName() {
         return getMetaName() + ".name";
     }
 
+    public <T> void addNotifiedInput(T input) {
```

Comment:
I am not understanding the logic here?

Why do items use the top level class IItemHandlerModifiable
while the fluids use the one subclass FluidTank in one place and another subclass NotifiableFluidTank in another.

Why don't the fluids use the IFluidHandler top level class?